<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons/Resource.qrc">
    <normaloff>:/icon/TE.png</normaloff>:/icon/TE.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(223, 223, 223);</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item alignment="Qt::AlignLeft">
       <widget class="QTreeView" name="treeView">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(210, 218, 226);</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
        </property>
        <property name="animated">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Untitled</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTextEdit" name="textEdit">
            <property name="toolTip">
             <string/>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(223, 223, 223);</string>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhMultiLine</set>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
            </property>
            <property name="tabChangesFocus">
             <bool>false</bool>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionRedo"/>
    <addaction name="actionUndo"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_TextEdit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>Zoom</string>
     </property>
     <property name="icon">
      <iconset resource="Icons/Resource.qrc">
       <normaloff>:/res/ZoomIn.png</normaloff>:/res/ZoomIn.png</iconset>
     </property>
     <addaction name="actionZoom_In"/>
     <addaction name="actionZoom_Out"/>
    </widget>
    <addaction name="menuZoom"/>
   </widget>
   <widget class="QMenu" name="menuFont">
    <property name="title">
     <string>Adjust</string>
    </property>
    <addaction name="actionSelect_Font"/>
    <addaction name="separator"/>
    <addaction name="actionItalic"/>
    <addaction name="separator"/>
    <addaction name="actionCenter"/>
    <addaction name="actionLeft_Align"/>
    <addaction name="actionRight_Align"/>
    <addaction name="actionJustified"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuFont"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_Font"/>
   <addaction name="actionItalic"/>
   <addaction name="separator"/>
   <addaction name="actionCenter"/>
   <addaction name="actionLeft_Align"/>
   <addaction name="actionRight_Align"/>
   <addaction name="actionJustified"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="separator"/>
   <addaction name="actionAbout_TextEdit"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNew">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/newfile.png</normaloff>:/res/newfile.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/open.png</normaloff>:/res/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/save.png</normaloff>:/res/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/save as.png</normaloff>:/res/save as.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/cut.png</normaloff>:/res/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/Copy.png</normaloff>:/res/Copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/Paste.png</normaloff>:/res/Paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/redo.png</normaloff>:/res/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/undo.png</normaloff>:/res/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionAbout_TextEdit">
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/info.png</normaloff>:/res/info.png</iconset>
   </property>
   <property name="text">
    <string>About TextEdit</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/ZoomIn.png</normaloff>:/res/ZoomIn.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/ZoomOut.png</normaloff>:/res/ZoomOut.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionSelect_Font">
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/Font-Book-icon.png</normaloff>:/res/Font-Book-icon.png</iconset>
   </property>
   <property name="text">
    <string>Select Font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/Editing-Italic-icon.png</normaloff>:/res/Editing-Italic-icon.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/align-center (1).png</normaloff>:/res/align-center (1).png</iconset>
   </property>
   <property name="text">
    <string>Center Align</string>
   </property>
  </action>
  <action name="actionLeft_Align">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/left-alignment.png</normaloff>:/res/left-alignment.png</iconset>
   </property>
   <property name="text">
    <string>Left Align</string>
   </property>
  </action>
  <action name="actionRight_Align">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/align-right.png</normaloff>:/res/align-right.png</iconset>
   </property>
   <property name="text">
    <string>Right Align</string>
   </property>
  </action>
  <action name="actionJustified">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/Resource.qrc">
     <normaloff>:/res/justify-text.png</normaloff>:/res/justify-text.png</iconset>
   </property>
   <property name="text">
    <string>Justified</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icons/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
